modules/nf-core/fastqc/tests/main.nf.test

```diff
@@ -21,12 +25,12 @@ nextflow_process {
         }
 
         then {
-            assert process.success
-            assert process.out.html.get(0).get(1) ==~ ".*/test_fastqc.html"
-            assert path(process.out.html.get(0).get(1)).getText().contains("<tr><td>File type</td><td>Conventional base calls</td></tr>")
-            assert process.out.zip.get(0).get(1) ==~ ".*/test_fastqc.zip"
+            assertAll (
+            { assert process.success },
+            { assert snapshot(process.out.html).match("html") },
+            { assert snapshot(process.out.versions).match("versions") },
+            { assert process.out.zip.get(0).get(1) ==~ ".*/test_fastqc.zip" }
```

Comment:
What does this do?

Reply:
1. check if the process has succeeded
2. insert process out `html` into snapshot
3. 2. insert process out `versions` into snapshot 
4. check if the process out `zip` produces a zip file